src/adapter/tstype.ts

```diff
@@ -10,7 +10,7 @@ interface NetworkOptions {
 interface SerialPortOptions {
     baudRate: number;
     rtscts: boolean;
-    path: string;
+    path?: string;
```

Comment:
How can this be optional?

Reply:
That seems to be what the stack further down accepts. Though I doubt it would do anything useful.

But I see now that my initial PR got amended with stuff I pushed to master ... that made a bit of a mess. I was just trying to get the library to work in another typescript project. Sorry.